src/es/components/controllers/fetchCss/FetchCss.js

```diff
@@ -158,6 +158,7 @@ export default class FetchCss extends Shadow(WebWorker()) {
         style = await this.webWorker(FetchCss.cssNamespaceToVar, style, fetchCSSParam.namespace)
       }
     }
+    if (fetchCSSParam.replaces) style = await fetchCSSParam.replaces.reduce((style, replace) => this.webWorker(FetchCss.replace, style, replace.pattern, replace.flags, replace.replacement), style)
```

Comment:
This line is hard to understand but it does the work.

Reply:
yes, reduce itself is kinda hard to read. any better proposals?

Reply:
I can't think of anything better atm 🤔 
```javascript
if (fetchCSSParam && fetchCSSParam.replaces) {
  style = await fetchCSSParam.replaces.reduce((style, replace) => this.webWorker(FetchCss.replace, style, replace.pattern, replace.flags, replace.replacement), style);
}
```